docs/tutorials/FlowTutorial.md

```diff
@@ -0,0 +1,2176 @@
+based on library(This is Nangate45 specific):
+```
+[INFO IFP-0030] Inserted 1 tiecells using LOGIC0_X1/Z.
+[INFO IFP-0030] Inserted 1 tiecells using LOGIC1_X1/Z.
+```
+
+### Macro or Cell Placement
+
+#### Macro Placement
+
+In this section, you will explore various placement options for macros
+and standard cells and study the impact on area and timing.
+
+Refer to the following built-in example [here](https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/gpl/test/macro01.tcl)
+to learn about macro placement.
+
+Placement density impacts how widely standard cells are placed in the
+core area. To view this in OpenROAD GUI run the following command(s) in the terminal in OpenROAD tool root directory:
+
+```
+cd src/gpl/test/
+openroad -gui
+```
+
+In the `Tcl Commands` section of GUI
+
+```
+source macro01.tcl
+```
+
+Read the resulting macro placement with a complete core view:
+
+![Macro Placement full view](./images/macro_place_full_view.png)
+
+Zoomed view of cell placement:
+
+![Macro Placement Zoomed view](./images/macro_place_close_view.png)
+
+Reduce the placement density and observe the impact on placement, by
+running below command in `Tcl Commands` section of GUI:
+
+```
+global_placement -density 0.6
+```
+
+Read the resulting macro placement with a complete core view:
+
+![Placement Density 60](./images/placement_density_06_full.png)
+
+Zoomed view of cell placement:
+
+![Placement density zoomed](./images/placement_density_06_zoomed.png)
+
+##### Macro Placement With Halo Spacing
+
+<!-- Explore macro placement with halo spacing, refer to the example
+[here](../../src/mpl/test/gcd_mem1_02.tcl). -->
+
+Launch GUI by running the following command(s) in the terminal in OpenROAD tool root directory:
+```
+cd src/mpl/test
+openroad -gui
+```
+
+In the `Tcl Commands` section of GUI:
+
+```
+source helpers.tcl
+read_liberty Nangate45/Nangate45_typ.lib
+read_liberty Nangate45/fakeram45_64x7.lib
+read_lef Nangate45/Nangate45.lef
+read_lef Nangate45/fakeram45_64x7.lef
+read_def gcd_mem1.def
+read_sdc gcd.sdc
+```
+
+DEF file without halo spacing
+
+![gcd without halo spacing](./images/without_halo.png)
+
+Now increase the halo width for better routing resources.
+
+In the `Tcl Commands` section of GUI:
+
+```
+macro_placement -halo {2.0 2.0}
+```
+
+DEF file with macro moved 2.0 micron from the left side of the core edge.
+
+![gcd with halo spacing](./images/with_halo.png)
+
+Run `global_placement` to align standard cells and macros with updated
+macro spacing.
+
+```
+global_placement
+```
+
+View the resulting DEF file as shown below
+
+![Aligned view](./images/halo_aligned.png)
+
+##### Macro Placement With Channel Spacing
+
+Now we will study how macro placement with channel spacing works.
+
+<!-- If the design has more than one macro, it is important to provide halo and
+channel spacing to provide enough space for routing. Refer to the
+following example [here](../../src/mpl/test/gcd_mem3_03.tcl). -->
```

Comment:
@vvbandeira this script is not there anymore as well

Reply:
same update required.